lib/src/main/java/com/basistheory/android/view/TextElement.kt

```diff
@@ -66,6 +69,17 @@ class TextElement : FrameLayout {
             input.hint = value
         }
 
+    val isMaskComplete: Boolean?
+        get() = maskWatcher?.isDone
+
+    val isMaskTouched: Boolean?
+        get() = maskWatcher?.isDirty
```

Comment:
Do we want to expose these properties? It looks like web elements doesn't have an equivalent exposed. I can see value in knowing whether the input is complete, but what value is there in exposing `isMaskTouched`?

Reply:
`isMaskTouched` came about from the [formik](https://formik.org/docs/api/formik#dirty-boolean) world which I feel may be useful to perform validation on focus/blur events.

Reply:
Removed